test/fixer/sqs_client_test.rb

```diff
@@ -0,0 +1,62 @@
+# encoding: utf-8
+
+require 'minitest_helper'
+
+describe Fixer::SqsClient do
+  let(:client) { Fixer::SqsClient.new }
+
+  let(:job_params) do
+    {
+      job: {
+        job_type: 'test',
+        priority: 1,
+        retry_max: 10,
+        retry_delay: 300,
+        tasks: [
+          sequence: {
+            tasks: [
+              {
+                task_type: 'echo',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.